java/src/main/java/com/logicalclocks/hsfs/TrainingDataset.java

```diff
@@ -501,37 +331,6 @@ public List<Object> getServingVector(Map<String, Object> entry) throws SQLExcept
     return vectorServer.getFeatureVector(this, entry);
   }
 
-  /**
-   * Retrieve feature vector from online feature store.
-   *
-   * @param entry Map object with kes as primary key names of the training dataset features groups and values as
-   *              corresponding ids to retrieve feature vector from online feature store.
-   * @param external If true, the connection to the online feature store will be established using the hostname
-   *                 provided in the hsfs.connection() setup.
-   * @return list of feature values sorted according to provided primary keys.
-   * @throws SQLException SQLException
-   * @throws IOException IOException
-   * @throws FeatureStoreException FeatureStoreException
-   * @throws ClassNotFoundException ClassNotFoundException
-   */
-  @JsonIgnore
-  public List<Object> getServingVector(Map<String, Object> entry, boolean external)
-      throws SQLException, FeatureStoreException, IOException, ClassNotFoundException {
-    return vectorServer.getFeatureVector(this, entry, external);
-  }
-
-  @JsonIgnore
-  public List<List<Object>> getServingVectors(Map<String, List<Object>> entry)
-          throws SQLException, FeatureStoreException, IOException, ClassNotFoundException {
-    return  vectorServer.getFeatureVectors(this, entry);
-  }
-
-  @JsonIgnore
-  public List<List<Object>> getServingVectors(Map<String, List<Object>> entry, boolean external)
-          throws SQLException, FeatureStoreException, IOException, ClassNotFoundException {
-    return vectorServer.getFeatureVectors(this, entry, external);
-  }
-
   /**
```

Comment:
maybe we shouldnt duplicate this doc string in multiple places, there already are inconsistencies. (here we have 2X `and` in row)